app/models/reading_collection.rb

```diff
@@ -3,5 +3,15 @@ class ReadingCollection < ApplicationRecord
   has_many :reading_collection_articles, dependent: :destroy
   has_many :articles, through: :reading_collection_articles
 
-  validates :name, :user_id, presence: true
+  validates :name, :user_id, presence: { presence: true }
+  validates :slug, presence: true, format: /\A[0-9a-z\-_]*\z/, uniqueness: { scope: :user_id }
```

Comment:
Where does the format come from? Interesting syntax

Reply:
I found it in other models in the app! Trying to be consistent.